contrib/labstack/echo.v4/echotrace.go

```diff
@@ -47,7 +49,15 @@ func Middleware(opts ...Option) echo.MiddlewareFunc {
 			defer span.Finish()
 
 			// pass the span through the request context
-			c.SetRequest(request.WithContext(ctx))
+			req := request.WithContext(ctx)
+			c.SetRequest(req)
+
+			if appsec.Enabled() {
```

Comment:
Can we apply this pattern everywhere now? Just for being consistent.

Reply:
I don't see other spots where it's applicable. I replaced `Status()` by `Enabled()` so that it would be applied everywhere `Status()` was used.

Reply:
`contrib/internal/httputil/trace.go`

Reply:
👍
Done in https://github.com/DataDog/dd-trace-go/pull/1042/commits/c5b4fb03d4d7de82663b66260b1ece5898ed2e7e

I'll add a proper support for global service entry span tags in a separate PR to properly have the `_dd.appsec.enabled: 0` case. The most important part is having it when it's `1` for now so that the service appears in the service list of the appsec dashboard.